model/roles.go

```diff
@@ -854,3 +854,39 @@ func (r *Role) IsDevRole() bool {
 	}
 	return false
 }
+
+// GetFileSignature returns the SHA1 of a file based on its path and contents
```

Comment:
Right. I used used in earlier revisions of the code and then it fell by the wayside.
Will remove.